samples/.eslintrc.yml

```diff
@@ -1,3 +1,4 @@
 ---
 rules:
   no-console: off
+  no-missing-require: off
```

Comment:
We shouldn't need this

Reply:
Because this is a sample app, the package.json is nested and wasn't getting installed properly by the CI so Ben Coe suggested it for this case.

Reply:
Got it! Done